bots/lambda_with_admin_privileges.py

```diff
@@ -0,0 +1,60 @@
+"""
+## lambda_with_admin_privileges
```

Comment:
Look for more straight forward name, you can advise David and Guy for bot documentation.

Reply:
change the name of the bot to lambda_detach_blanket_permissions(guy advised)